raiden-ts/src/messages/utils.ts

```diff
@@ -3,6 +3,7 @@ import { Signer } from 'ethers';
 import { keccak256, RLP, verifyMessage } from 'ethers/utils';
 import { arrayify, concat, hexlify } from 'ethers/utils/bytes';
 import { HashZero } from 'ethers/constants';
+import logging from 'loglevel';
```

Comment:
```suggestion
import { Logger } from 'loglevel';
```

Reply:
This can't be. Notice we use `logging` (the root logger) in the `signMessage` utils (as default parameter), so we need the whole default export of `loglevel`, which is the root logger.